packages/react-core/src/components/Brand/examples/BrandBasic.tsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 import { Brand } from '@patternfly/react-core';
-import pfLogo from '../../assets/pfLogo.svg';
+import pfLogo from '../../assets/PF-HorizontalLogo-Color.svg';
+import pfLogoDark from '../../assets/PF-HorizontalLogo-Reverse.svg';
 
 export const BrandBasic: React.FunctionComponent = () => (
-  <Brand src={pfLogo} alt="Patternfly Logo" widths={{ default: '200px', md: '400px', xl: '600px' }} />
+  <>
+    <div className="show-light">
+      <Brand src={pfLogo} alt="Patternfly Logo" widths={{ default: '200px', md: '400px', xl: '600px' }} />
+    </div>
+    <div className="show-dark">
+      <Brand src={pfLogoDark} alt="Patternfly Logo" widths={{ default: '200px', md: '400px', xl: '600px' }} />
```

Comment:
For the alt text in these examples, can we just omit the word "logos", so just `alt="PatternFly"`.

Reply:
sounds good, done!